flocker/node/test/istatechange.py

```diff
@@ -17,9 +17,9 @@
 from pyrsistent import PClass, field
 from characteristic import attributes
 
-from twisted.trial.unittest import SynchronousTestCase
 from twisted.internet.defer import succeed
 
+from flocker.testtools import TestCase
```

Comment:
:question: Relative imports, for when we finally decide to pick a non-silly name :stuck_out_tongue_winking_eye: